.github/workflows/auto-update-i18n.yml

```diff
@@ -15,6 +15,8 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
         with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
i don't think you need any input here, i don't see any in the other CI

Reply:
it is needed for the add-and-comit action. They state that if working with PRs you need to add that. Thats why the last run I did failed. 

It is also in ic-js/.github/workflows/docs.yml

Reply:
Ah understood! thanks for the explanation